docker-compose.yaml

```diff
@@ -83,7 +83,8 @@ services:
       context: ./prometheus
       dockerfile: Dockerfile
       args:
-        TARGETS: ${PROMETHEUS_TARGETS:-jellyfish1:9568,jellyfish2:9568}
+        JELLYFISH_TARGETS: ${PROMETHEUS_TARGETS:-jellyfish1:9568,jellyfish2:9568}
+        NODE_TARGETS: '172.17.0.1:9100'
```

Comment:
Node-exporter, to get access to more statistics regarding the network, must have access to the OS network; that's why it is run in `network_mode: host`. When it is run in this mode, it can't be referred by other services via `service_name,` as other containers don't run in `network_mode: host`. Furthermore, They don't have access through localhost to services run on the host OS. To allow this, docker added a special IP address that allows access services run on the host OS and this IP address is `172.17.0.1`.
So to summarize `172.17.0.1` in docker containers allows access services run on `127.0.0.1` in the host OS.

Reply:
Could we please add a short comment?